apps/web/config/env.server.ts

```diff
@@ -2,6 +2,7 @@ import { EthNetworkEnum } from "@/common/eth-network-enum";
 import { addressSchema } from "@repo/common/schemas";
 import { createEnv } from "@t3-oss/env-core";
 import { z } from "zod";
+import "dotenv/config";
```

Comment:
If this is now a production library, it should be moved from `devDependencies` to `dependencies` in package.json

Reply:
Makes sense. This was already needed to run npm start, so it was already not a dev dependecny.